### DISCUSSION TITLE
760

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#927] [Comments]
# 760

아래 글은 오늘 팀 내부에 공유한 글입니다. 팀을 떠난 후에도 다시 볼 일이 있을것 같아 블로그에도 남깁니다. 오늘 추천 받은 영상인데, 그간 저희 인프랩 조직 내에서 계속 이야기하던 주제와 맞물려서 공유드립니다. (전) 구글 인사담당자셨던 황성현님의 인터뷰를 보면 다음과 같은 내용이 나옵니다. "엄청난 스트레스를 받는 펀드 매니저들 중 최고 경영층까지 갔던 사람들의 공통된 특징"을 보니 어렸을때부터 12학년 (고3) 까지 팀 스포츠를 했다는 것인데요. 그냥 만년 후보 선수로 벤치를 지키고 있더라도, Being part of a winning team - 이기는 팀의 맴버였던 경험이 있던 사람이 끝까지 살아남을 수 있는 비결이라는 것입니다. 즉, 내가 4번 타자였던게 중요한게 아니라 우리 팀이 1등한게 ..

https://jojoldu.tistory.com/760

**Comment by Invidam** [comment]
어렸을 적 성공경험을 경험한 사람과 아닌 사람들 간의 가치관 차이가 있다고 들은 기억이 있는데, 비슷한 맥락인 듯 하네요 ㅎㅎ

잘 읽었습니다 👍
